lib/extensions/base.py

```diff
@@ -128,8 +130,8 @@ def no_elements_error(self):
         else:
             inkex.errormsg(_("There are no objects in the entire document that Ink/Stitch knows how to work with.") + "\n")
 
-        inkex.errormsg(_("Ink/Stitch only knows how to work with paths.  It can't work with objects like text, rectangles, or circles.") + "\n")
-        inkex.errormsg(_("Tip: select some objects and use Path -> Object to Path to convert them to paths.") + "\n")
+        inkex.errormsg(_("Tip 1: Check if the object has the correct color settings applied to it (Object -> Fill and Stroke...).") + "\n")
```

Comment:
How do color settings make Ink/Stitch think there's nothing to embroider?

Reply:
Hm, it's a good question. I forgot. My first guess was, I had the idea from here: https://www.w3schools.com/graphics/svg_polyline.asp while testing polylines. But this isn't right and actually an issue, we should have a look upon separately (colors are set by their name).

I think I should remove this comment for now. Color settings shouldn't cause an error like this.